app/handlers/profiles/settings/username.go

```diff
@@ -59,7 +60,7 @@ func NameChangeSubmitHandler(c *gin.Context) {
 			sessions.AddMessage(c, msg.ErrorMessage{lu.T(c, "Something went wrong.")})
 			sessions.GetSession(c).Save()
 
-			fmt.Println("Error adding to user notes: ", logErr.Error())
+			slog.Error("error", "Error adding to user notes: ", logErr.Error())
```

Comment:
why "error" here?

Reply:
docs say the call spec is `func Error(msg string, args ...any)` https://pkg.go.dev/golang.org/x/exp/slog#Error